packages/flutter/lib/src/animation/curves.dart

```diff
@@ -522,6 +530,11 @@ class Curves {
   /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_ease_in.mp4}
   static const Cubic easeIn = Cubic(0.42, 0.0, 1.0, 1.0);
 
+  /// A cubic animation curve that starts starts slowly and ends linearly.
+  ///
+  /// The symmetric animation to [linearToEaseOut].
```

Comment:
These ones seem better described. Remember to add an animation movie here and in the class doc. @dnfield has this fresh in his mind so he might be able to help.